src/hooks/use-fetch-wpcom-sites.tsx

```diff
@@ -106,7 +123,7 @@ export const useFetchWpComSites = ( connectedSiteIds: number[] ) => {
 					path: `/me/sites`,
 				},
 				{
-					fields: 'name,ID,URL,plan,is_wpcom_staging_site,is_wpcom_atomic,options',
+					fields: 'name,ID,URL,plan,is_wpcom_staging_site,is_wpcom_atomic,options,jetpack',
```

Comment:
I added only this extra check but we could also check for `jetpack_connection_active_plugins` that JP sites have but I did not find it necessary in my tests as the `jetpack` check seemed to be enough